src/content/community/code-of-conduct/index.md

```diff
@@ -0,0 +1,75 @@
+
+### Scope
+
+This Code of Conduct applies to all [ethereum.org](http://ethereum.org) spaces (such as Github, Discord, Figma Crowdin, Twitter and other online platforms), and it also applies when the community is represented in real-world public spaces such as at meetups, conferences and events.
+
+### Our standards
+
+Examples of behaviour that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting and/or empathetically providing constructive criticism
+- Acting calmly and professionally when resolving conflicts or disagreements
+- Showing empathy and tolerance towards other community members
+- Encouraging and amplifying new voices in the community
+
+Examples of unacceptable behavior by participants include:
+
+- Physical violence, threatening physical violence or encouraging of physical violence of any kind
+- Using sexualized language or imagery or imposing unwelcome sexual attention
+- Impersonating another individual or otherwise dishonestly claiming affiliation with some individual or organization.
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Harrassing other community members in public or private channels
+- Publishing others’ private information, such as a physical or electronic addresses, without explicit permission
+- Social engineering, scamming or otherwise manipulating other community members
+- Promoting investments, tokens, projects or anything else for personal monetary or non-monetary gain
+- Spamming servers with off-topic content
+- Disregarding requests or warnings from community moderators
+- Engaging in other conduct which could reasonably be considered inappropriate in a professional setting
+
+### Reporting
+
+Violations of the code of conduct will normally be visible to the community as we try to do everything in open, public channels, allowing community members to self police.
+
+However, if something happens that you feel needs attention, you can raise it with someone who has a moderation role (e.g. discord guide) so that they can help investigate and execute the appropriate response.
+
+When reporting, please try to include as much detail as you can, including specific examples and timestamps where possible. This will help to ensure a fair outcome.
```

Comment:
```suggestion
When reporting, please include as much detail as possible, including specific examples and timestamps. This will help to ensure a fair outcome.
```